templates/example_qc_metric_eval.ipynb

```diff
@@ -3,6 +3,7 @@
   {
```

Comment:
<p>What is the point of this flag? The cells below don't contain any other code, so either the user runs them and generates the graphs, or doesn't run them and doesn't generate the graphs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Just an additional control flow flag, I'll remove it.</p>